.github/workflows/main.yml

```diff
@@ -40,6 +54,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Store Gradle cache
```

Comment:
Duplicate job? (already defined above)

Reply:
This one is for the andorid-tests job, while the one above is for java-tests